Tasks/AzurePolicyManageV1/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
I suppose these other files are related to App Service task, perhaps remove them until loc hand back happens?